dt-groups/groups.js

```diff
@@ -75,11 +75,20 @@ jQuery(document).ready(function($) {
       if( member.leader ){
         leaderHTML = `<i class="fi-foot small leader"></i>`
       }
+      const contactStatusHTML = ( member.data && member.data.overall_status ) 
```

Comment:
This section needs a bit of love from window.lodash.escape()

Reply:
doh :man_facepalming:

Reply:
resolved at 1c0dc67b